src/features/defi/providers/foxy/components/FoxyManager/FoxyManager.tsx

```diff
@@ -13,7 +13,7 @@ import { FoxyDeposit } from './Deposit/FoxyDeposit'
 import { FoxyOverview } from './Overview/FoxyOverview'
 import { FoxyWithdraw } from './Withdraw/FoxyWithdraw'
 
-enum FoxyPath {
+export enum FoxyPath {
```

Comment:
Should we move this enum to common file with only types/enums/constants. If another file only needs to use the enum, there's no point in importing the entire manager with it.

Reply:
TIL! https://262.ecma-international.org/6.0/#sec-abstract-module-records
Although from looking at this, the module should have already been evaluated once and thus the whole module won't be evaluated a second time if I got this right?

Reply:
@theoboldfrazier moved to `FoxyCommon.ts`